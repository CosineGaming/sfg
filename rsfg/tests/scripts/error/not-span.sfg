fn main()
	if !"str"
		panic()

