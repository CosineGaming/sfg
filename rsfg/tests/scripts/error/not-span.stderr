[ERROR] mismatched type, expected bool at 2:6, got str at 2:6